tests/prepared_statement_tests/update_all.cpp

```diff
@@ -39,15 +39,6 @@ TEST_CASE("Prepared update all") {
 
     {  //  by val
         auto statement = storage.prepare(update_all(set(assign(&User::name, conc(&User::name, "_")))));
-        using Statement = decltype(statement);
```

Comment:
why did you remove these static parts of ut?

Reply:
They are covered by the tests already in tests/bindables.cpp and the new "set assign" section in [tests/node_tuple.cpp](https://github.com/fnc12/sqlite_orm/pull/1241/files/9813ec16425e3e4f070a7c22ddafb5cfa16d792f?diff=split&w=0#diff-567fe4ab260b49c101b491d47de299ea1da4e62338918924d86e98c91b8d51fb).